packages/suite-desktop/src-electron/libs/processes/TorProcess.ts

```diff
@@ -1,20 +1,42 @@
-import fetch from 'node-fetch';
-
+import path from 'path';
 import BaseProcess, { Status } from './BaseProcess';
 
-// 9050 is the default port of the tor process.
-export const DEFAULT_ADDRESS = '127.0.0.1:9050';
+import { TorController } from '@trezor/request-manager';
+
+interface TorConnectionOptions {
```

Comment:
I think the `TorConnectionOptions` interface could be defined in one place only (and maybe even `TorControlPortOptions` which is almost the same).

Reply:
Done c46cb393222f991634b270916408c038976d673e